pkg/sqlite/load.go

```diff
@@ -620,7 +622,7 @@ func (s *sqlLoader) addPackageChannels(tx *sql.Tx, manifest registry.PackageMani
 		}
 
 		channelEntryCSVName := c.CurrentCSVName
-		depth := 1
+		depth := startDepth + 1
```

Comment:
Would be nice to add a comment to explain that we already add the currentCSV at depth = 0 so now the depth is at 1. Honestly, this entire section of code is lacking of comments but it is not the main purpose of this PR to address that.

Reply:
I'll add a comment about this line, but I don't think I'm prepared to add comments for the entire function (nor should we in this PR since this function is functionally identical, since `startDepth` is 0)